adoptadog-blog.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt A Dog Blog</title>
+    <link rel="stylesheet" type="text/css" href="styles/adoptadog-stylesheet.css">
+    <script src="https://kit.fontawesome.com/015d63ce65.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css?family=Courgette%7CPT+Sans:700&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+
+    <header>
+      <div class="grid-container">
+        <div class="grid-item-1">
+          <img src="images/adoptadog-logo.svg" alt="Adopt A Dog Logo" style="min-width:20rem; max-width:60%;">
+          <h1 class="header-h1">Seattle's premiere dog adoption agency</h1>
+        </div>
+        <div class="grid-item-2">
+          <img src="images/cart-icon.svg" alt="Click to go to checkout" style="min-width: 2rem; max-width:5%;">$1234.56
+          <p><a href="adoptadog-checkout.html"><img src="images/checkout-button.jpg" alt="Click to go to checkout" style="min-width: 7.5rem; max-width:30%;"></a>
+        </div>
+      </div>
+    </header>
+
+    <!-- Create navigation pane -->
+    <nav class="nav-header">
+      <a class="nav-link" href="adoptadog-index.html">Home / </a>
+      <a class="nav-link" href="adoptadog-dogs.html">Dogs / </a>
+      <a class="nav-link" href="adoptadog-blog.html">Blog / </a>
+      <a class="nav-link" href="adoptadog-checkout.html"> Checkout</a>
+    </nav>
+
+    <h1 style="padding:1em 10%;">Adoptadog Blog</h1>
+
+    <script src="./js/adopt-a-dog-blog.js"></script>
+
+    <div class="grid-container" id = "blogContainer1">
```

Comment:
You Should not have elements hardcoded into the html, but generate all the html using JS.